### PR TITLE
remove logstash/pipeline

### PR DESCRIPTION
**Pull request by kaisecheng**
Fixed: https://github.com/elastic/logstash/issues/11236

This PR replace remove ruby pipeline

### CODE REVIEW

File: .ci/common/logstash.Dockerfile [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ARG ELASTIC_STACK_VERSION
 FROM docker.elastic.co/logstash/logstash:$ELASTIC_STACK_VERSION
-COPY --chown=logstash:logstash Gemfile /usr/share/plugins/plugin/
-COPY --chown=logstash:logstash *.gemspec /usr/share/plugins/plugin/
+USER logstash
+COPY --chown=logstash:logstash Gemfile /usr/share/plugins/plugin/Gemfile
+COPY --chown=logstash:logstash *.gemspec VERSION* version* /usr/share/plugins/plugin/
 RUN cp /usr/share/logstash/logstash-core/versions-gem-copy.yml /usr/share/logstash/versions.yml
-ENV PATH="${PATH}:/usr/share/logstash/vendor/jruby/bin"
+# NOTE: since 8.0 JDK is bundled as part of the LS distribution under $LS_HOME/jdk
```
Comment (andsel) [inline .ci/common/logstash.Dockerfile:7]: It's from `7.10` that's bundled in the distribution

File: .ci/docker-setup.sh [hunk 2]
```diff
@@ -8,56 +8,60 @@
...
   # remove starting and trailing double quotes
   ELASTIC_STACK_RETRIEVED_VERSION="${ELASTIC_STACK_RETRIEVED_VERSION%\"}"
   ELASTIC_STACK_RETRIEVED_VERSION="${ELASTIC_STACK_RETRIEVED_VERSION#\"}"
   echo "Translated $ELASTIC_STACK_VERSION to ${ELASTIC_STACK_RETRIEVED_VERSION}"
   export ELASTIC_STACK_VERSION=$ELASTIC_STACK_RETRIEVED_VERSION
 fi
 
- echo "Testing against version: $ELASTIC_STACK_VERSION"
-
- if [[ "$ELASTIC_STACK_VERSION" = *"-SNAPSHOT" ]]; then
```
Comment (andsel) [inline .ci/docker-setup.sh:43]: I've not a such a deep knowledge of docker, but `docker pull` is the same as this `curl` + `docker load` ?

**Review by andsel — COMMENTED** [review]
I've left a note and a question

File: .ci/common/logstash.Dockerfile (hunk 1 above)
Comment (kaisecheng) [reply .ci/common/logstash.Dockerfile:7]: it is copied from .ci/Dockerfile I can change it to 7.10

File: .ci/docker-setup.sh (hunk 2 above)
Comment (kaisecheng) [reply .ci/docker-setup.sh:43]: `docker pull` pull the image from the registry while `docker load` import image from STDIN .tar.gz

**Review by andsel — APPROVED** [review]
LGTM

**Pull request merged**